src/bundle/Resources/views/themes/admin/content_type/field_types.html.twig

```diff
@@ -233,7 +233,7 @@
             </button>
             <button
                 type="button"
-                class="ezselection-settings-option-remove ez-font-icon ez-button btn btn-danger ez-button-delete pure-button"
+                class="ezselection-settings-option-remove ez-font-icon ez-button btn btn-primary ez-button-delete pure-button"
```

Comment:
I'm not sure if this should be our primary color. Does the remove option is the primary action?

Reply:
All our modal "Delete" buttons, trash icons etc have primary color so answering your question, it seems so